packages/alert/src/lib/Alert.tsx

```diff
@@ -3,15 +3,20 @@ import { Alert as MUIAlert } from '@mui/material';
 import type { AlertProps as MUIAlertProps } from '@mui/material';
 import { InfoCircleIcon, WarningCircleIcon, SuccessCircleIcon } from '@availity/mui-icon';
 
-export type AlertProps = Omit<MUIAlertProps, 'variant'>;
+export type AlertProps = Omit<MUIAlertProps, 'variant'> & {
+  severity?: 'success' | 'info' | 'warning' | 'error';
+};
 
-const iconMapping = {
-  error: <WarningCircleIcon aria-hidden={false} titleAccess="error" />,
-  info: <InfoCircleIcon aria-hidden={false} />,
-  success: <SuccessCircleIcon aria-hidden={false} />,
-  warning: <WarningCircleIcon aria-hidden={false} sx={{ color: 'warning.dark' }} />,
+const getIconMapping = (color: 'success' | 'info' | 'warning' | 'error' | undefined) => {
```

Comment:
I don't believe we want users using the color prop, they should only be using severity iirc. Also I don't believe the other icon colors in the specs use main.